joomla/joomla-cms
# [5.1] Convert mod_loginsupport to service provider

### Summary of Changes

Converts the login support module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_loginsupport published and configured
2. You should see module output on administrator login page

![mod_loginsupport](https://github.com/joomla/joomla-cms/assets/59166858/ac91659b-5cd0-41d9-9a0b-50292836f32a)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
